### DISCUSSION TITLE
Good way to secure the stock battery?

### DISCUSSION THREAD

**Discussion by quitmeyer** [PiSupply/PiJuice#1070] [General]
Our pijuice is being used as part of a piece of scientific field equipment. It is connected to  a pi4 and also has a relay hat stacked on top of it.

One challenge we have discovered is that when we are carrying these devices over rough terrian they can shake A LOT. and then the little stock batteries can sometimes pop out, or partially dislodge. This is a problem because it stores the RTC commands on it and if it loses power, that's bad!

Does anyone have good suggestions to really secure that battery in place?

my basic idea was to use some zip ties or something like this, but im guessing someone probably had a more elegant idea already

![image](https://github.com/PiSupply/PiJuice/assets/742627/31132f45-ba6b-47b3-a39b-b48b7787b470)

**Comment by BrianWood67** [comment]
We envisaged a similar problem with our in-vehicle application.
The successful solution is to remove the wrapper from the battery. Apply JB weld between the battery and the board. Use zipties to secure in place while it dries overnight.

**Reply by quitmeyer** [reply]
great thinking!

**Reply by ArjenR49** [reply]
Eventually you will have to replace the battery. What is JB weld? Some kind of glue?

**Reply by tvoverbeek** [reply]
Yes. See https://www.jbweld.com

**Reply by ArjenR49** [reply]
I would suggest some sort of elastic bumper block between battery and relay HAT on top of the PiJuice HAT rather than any sort of glue under the battery. You could glue the bumper to the battery, if you feel that is safer.
